recipe/recipe.yaml

```
@@ -1,15 +1,15 @@
 context:
   name: numba-cuda
   sdist_name: numba_cuda
-  version: "0.0.17"
+  version: "0.0.17.1"
```

Looks like we need this to workaround issue: https://github.com/prefix-dev/rattler-build/issues/1248

```suggestion
  version: "0.0.17.1"
  python_min: "3.9"
```